doc/content/getting-started/migrating-from-v2/major-changes.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Major Changes in The Things Stack
+weight: 10
+---
+
+Before getting started, we will discuss major breaking changes between {{% ttnv2 %}} and {{% tts %}}, along with some guidelines to make the migration process easier to manage.
+
+<!--more-->
+
+## Architecture
+
+Behind the scenes, {{% tts %}} is using a new architecture. See [Components]({{< ref "/reference/components" >}}) for a complete overview of the different components of {{% tts %}}.
+
+## LoRaWAN support
+
+{{% tts %}} requires the LoRaWAN MAC Protocol Version and Regional Parameters (LoRaWAN PHY version) to be set per end device. These default to `LoRaWAN 1.0.2` and `LoRaWAN Regional Parameters 1.0.2 Rev B` for end devices imported from {{% ttnv2 %}}, because this configuration is the most consistent with the V2.
+
+The LoRaWAN MAC settings are also configurable per end device. This means that all end devices need a frequency plan. Above, we used the `EU_863_870` frequency plan, but this needs to be replaced with the frequency plan corresponding to your region. A list of supported frequency plan IDs in [the lorawan-frequency-plans Github repository](https://github.com/TheThingsNetwork/lorawan-frequency-plans/blob/master/frequency-plans.yml).
+
+<!--
+TODO: https://github.com/TheThingsNetwork/lorawan-stack/issues/2421
+Add reference to docs after that is merged.
+-->
+
+Furthermore, {{% tts %}} brings full support for all LoRaWAN versions, as well as Class B and Class C end devices.
+
+## Gateway Coverage
+
+The Packet Broker enables peering between networks, so traffic received by one network (e.g. the public community network) but intended for a different network ({{% tts %}}) can be forwarded to and from that network. See the [Peering Guide]({{< ref "/integrations/peering" >}}) for details on Packet Broker and how to enable it for your network.
+
+With Packet Broker enabled, you can receive traffic on {{% tts %}} without the need to configure any of your gateways.
```

Comment:
This is only if the old network is also on Packet Broker. That is not the case for most V2 private networks. So I would claim this only when the gateways are connected to The Things Network public community network